frontend/README.md

```diff
@@ -39,3 +39,19 @@ If the standalone TypeScript plugin doesn't feel fast enough to you, Volar has a
    1. Run `Extensions: Show Built-in Extensions` from VSCode's command palette
    2. Find `TypeScript and JavaScript Language Features`, right click and select `Disable (Workspace)`
 2. Reload the VSCode window by running `Developer: Reload Window` from the command palette.
+
+## Setup Playwright for E2E Tests
```

Comment:
You could also add how to run just 1 specific test:

npm run test:e2e -- [testfile]

And also maybe add how to debug an e2e test:

npm run test:e2e -- [testfile] --debug